pyproject.toml

```diff
@@ -22,9 +22,7 @@ dependencies = [
 ]
```

Comment:
Not sure I get you, are you talking about Python version? Ruff version?

Reply:
I am talking about using ">=" instead of "==" in requirements. Is there any reason to do it ?

Reply:
@dtrckd Let us know if we can merge that one @dtrckd !

Reply:
Understood, good point.

Normally if I'm not wrong, the good practice would be to have a lock dependency file (`requirements.lock`, or `pdm.lock`, or `poetry.lock` - we could even use a `requirements.txt` file as the lock dependency file) commited along the main package file. Like NPM with `package.json` + `package-lock.json`.

Since AFAIK the team is using mostly `pip` which does not really manage the creation of a lock dependency file.
So we could pin version in `pyproject.toml` indeed.

I updated the file in this branch. Let me know what you think.

Reply:
Yes I think we good to go, I don't see any other problems as the others modifications are just style.

Reply:
I mainly wanted make sure the pre-commit was behaving the same way on your side.
Merging now